test/Sema/large_int_array.swift.gyb

```diff
@@ -0,0 +1,11 @@
+// RUN: %empty-directory(%t)
+// RUN: %gyb %s -o %t/large_int_array.swift
+// RUN: %target-swift-frontend -typecheck -verify %t/large_int_array.swift
+
+% num_elements = 65537
+
+public let largeIntArray = [
```

Comment:
How long does it take to type-check this literal? Let’s remove `public` and add `: [Int]` since the test is really about the number of items in the array..

Reply:
Fortunately, quite fast (as compared to other cases from #60549):
```
time swiftc -frontend -c -o out.o -primary-file LargeIntArray.swift

________________________________________________________
Executed in    3.74 secs    fish           external
   usr time    3.54 secs    0.10 millis    3.54 secs
   sys time    0.17 secs    3.52 millis    0.17 secs
```

Removing `public` and making the type explicit does not change the runtime though

Reply:
How about this one?

```swift
let v = 0

let _ = [
% for i in range(num_elements):
  v,
% end
]
```

Reply:
Slightly faster:

```
________________________________________________________
Executed in    2.69 secs    fish           external
   usr time    3.69 secs    0.06 millis    3.69 secs
   sys time    0.84 secs    1.78 millis    0.84 secs
```